src/assets/scss/6-components/atoms/_text-input-icon.scss

```diff
@@ -0,0 +1,17 @@
+.c-text-input-icon {
+    position: relative;
+
+    .c-icon {
+        position: absolute;
+        left: 0;
+        top: calc(50% - 14px);
+        path {
+            fill: get-color-neutral("50");
```

Comment:
Not sure if it was ported, but I do remember making an `icon-fill` mixin at some point

Reply:
Yeah, much of the scss and meat of components are direct ports. I did some minor clean up as I could. Mostly spent time getting stories and some test rigging for many. As well as fixing up rendering needs (ie. icons)

Reply:
Yeah - no worries. Doesn't need to be 100% clean & polished for the first round 😝 Just a thought.